### DISCUSSION TITLE
[Feature] Implement image/video upscaler ?

### DISCUSSION THREAD

**Discussion by Nonobis** [immich-app/immich#1795] [Feature Request]
### The feature

Hello,

Maybe immich could use Video2X or if you prefer Implement a image/vidéo upscaler....

Personnaly, i have a lot of old pictures in immich i would love toi upscale.. Photos of kids or famille Taken with Bad resolution.

A settings ans a server job could be perfect.
Maybe a settings to only upscale to a max resolution.?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by arne182** [comment]
https://www.topazlabs.com/topaz-photo-ai
